virt/virt-4-10-release-notes.adoc

```diff
@@ -72,22 +72,28 @@ The SVVP Certification applies to:
 [id="virt-4-10-storage-new"]
 === Storage
 
+//CNV-16673 New CSI driver for HPP
+* You can use the xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Kubernetes Container Storage Interface (CSI) driver] with the hostpath provisioner (HPP) feature to configure local storage for your virtual machines. By creating an HPP custom resource that includes one or more storage pools, as well as creating a storage class for the CSI driver, you can run the HPP with less disruption to your existing {product-title} nodes and clusters.
+
 [id="virt-4-10-web-new"]
 === Web console
 
-// NOTE: no deprecated features in 4.10; commenting out these sections to retain for future use
-//[id="virt-4-10-deprecated-removed"]
-//== Deprecated and removed features
+// NOTE: Comment out deprecated and removed features (and their IDs) if not used in a release
+[id="virt-4-10-deprecated-removed"]
+== Deprecated and removed features
 
-//[id="virt-4-10-deprecated"]
-//=== Deprecated features
+[id="virt-4-10-deprecated"]
+=== Deprecated features
 
 //Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
 
+* In {VirtProductName} 4.10, the HPP Operator configures the Kubernetes Container Storage Interface (CSI) driver. The Operator also recognizes the existing (legacy) format of the custom resource. If you upgrade to {VirtProductName} 4.10, no action is required.
```

Comment:
Instead of using the note as is, I would suggest rephrasing this as:

`The legacy method of configuring local storage for virtual machines by using the Host Path Provisioner (HPP) Operator is being deprecated with this release. In {VirtProductName} {VirtVersion}, the HPP Operator configures the Kubernetes Container Storage Interface (CSI) driver. The Operator also supports the existing (legacy) format of the HPP custom resource. If you use the HPP Operator, plan to xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[create a storage class for the CSI driver] as part of your migration strategy.`

Reply:
Here are my thoughts:

* The legacy method of configuring local storage for virtual machines by using the Host Path Provisioner (HPP) Operator is being deprecated with this release. 
--- The HPP Operator is still used in 4.10. The difference is that the operator now uses a new, less disruptive, CSI driver in the implementation.
* Just making sure that VirtVersion won't result in wrong version numbers at some point. Do we run Deprecation warnings for several releases? If it stays in the notes, then it will read as if the default CSI implementation began in 4.11.

Maybe:

Beginning in {VirtProductName} 4.10, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. The Operator continues to support the existing (legacy) format of the HPP custom resource. In a future release, support for the legacy format will be deprecated. If you use the HPP Operator, plan to xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[create a storage class for the CSI driver] as part of your migration strategy.

WDYT?

Reply:
Changed to (in order to lead with the deprecation statement):

(I thought more about leaving out the storage class, which pairs with the format of he CR, but it didn't seem clear that way)

* In a future release, support for the legacy HPP custom resource, and the associated storage class, will be deprecated. Beginning in {VirtProductName} {VirtVersion}, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. The Operator continues to support the existing (legacy) format of the HPP custom resource and the associated storage class. If you use the HPP Operator, plan to xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[create a storage class for the CSI driver] as part of your migration strategy.